scylla/src/routing/partitioner.rs

```diff
@@ -75,15 +76,19 @@ impl PartitionerHasher for PartitionerHasherAny {
 }
 
 mod sealed {
-    pub(crate) trait Sealed {}
+    // This is a sealed trait - its whole purpose is to be unnameable.
+    // This means we need to disable the check.
+    #[allow(unknown_lints)] // Rust 1.70 doesn't know this lint
```

Comment:
:question: Is our MSRV still 1.70?

Reply:
Oh, it actually still is. That's because we haven't ported the MSRV bump from hackathon branch yet, right?

Reply:
Yes, it is part of my PR with CCM backport.